src/commands/environments/index.ts

```diff
@@ -16,9 +18,25 @@ export default class Environments extends Command {
   }];
 
   async run() {
-    const { args } = this.parse(Environments);
+    const { args, flags } = this.parse(Environments);
 
-    const { data: { rows: environments } } = await this.app.api.get(`/environments?q=${args.query || ''}`);
+    let account: Account | undefined = undefined;
+    if (flags.account) {
+      account = await AccountUtils.getAccount(this.app.api, flags.account);
+    }
+
+    // Prepare the query params to be reduced to a string
+    const query_params: any = {
+      q: args.query || '',
+    };
+    if (account) {
+      query_params['account_id'] = account.id;
+    }
+
+    // Reduces an object into a string of query parameters
+    const query_string = Object.entries(query_params).map(key_val => key_val.join('=')).join('&');
+
+    const { data: { rows: environments } } = await this.app.api.get(`/environments?${query_string}`);
```

Comment:
axios has an option for query params.

```ts
const params = {
  q: args.query || '',
  account_id: account?.id
}
const { data: { rows: environments } } = await this.app.api.get(`/environments`, { params });
```

Reply:
🤦 oof, I didn't realize that. Will make that change 👍